init.c

```diff
@@ -68,6 +69,27 @@
 #define MAX_PROC_STATUS (10)
 pid_t pv_pid;
 
+static void load_early_drivers(bool run_mdev)
```

Comment:
Wouldn't this be better in drivers.c? Also, woudln't it be better to use pv_config_get_system_mdev inside instead of using an argument?